hotfixes/wc-api-dev-email-order-url.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Provides a hotfix that points the admin order URL to WordPress.com instead of wp-admin.
+ *
+ * @since 0.8.2
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+function wc_api_dev_email_get_wpcom_order_link( $order_id ) {
+	$site_slug = preg_replace( '/^https?:\/\//', '', get_site_url() );
```

Comment:
Good catch and suggestion. I think it's a good idea to handle this now so we don't run into issues later.